lib/drivers/animasDriver.js

```diff
@@ -422,6 +442,52 @@ module.exports = function (config) {
     };
   };
 
+  var requestCalibration = function() {
+    return{
+      recordType: RECORD_TYPES.CGM_CALIBRATION,
+      parser : function (payload) {
+        var dt = decodeDate(struct.unpack(payload,4,'bbbb',['monthYear','day','hour','minute']));
```

Comment:
we decode the date in this same format twice in the new code - is it worth having a specialisation of the function that just assumes that format? maybe not but just wanted to put it out there